lib/bolognese/author_utils.rb

```diff
@@ -144,16 +144,23 @@ def get_affiliations(affiliations)
           affiliation_identifier_scheme = nil
           scheme_uri = nil
         else
+          scheme_uri = a["schemeURI"]
           if a["affiliationIdentifier"].present?
             affiliation_identifier = a["affiliationIdentifier"]
             if a["schemeURI"].present?
               schemeURI = a["schemeURI"].end_with?("/") ? a["schemeURI"] : a["schemeURI"] + "/"
             end
             affiliation_identifier = !affiliation_identifier.to_s.start_with?("https://") && schemeURI.present? ? normalize_id(schemeURI + affiliation_identifier) : normalize_id(affiliation_identifier)
+            # The normalize_id(affiliation_identifier) method currently discards affiliation identifiers that don't start with a URL, 
+            # for example: affiliation_identifier = "05bp8ka05".
+            # To address this issue, we are introducing the following change to handle such affiliation identifiers.
+            if a["affiliationIdentifierScheme"] == "ROR" && affiliation_identifier.nil?
+              scheme_uri = "https://ror.org"
+              affiliation_identifier = scheme_uri + "/" + a["affiliationIdentifier"]
+            end
           end
           name = a["__content__"].to_s.squish.presence
           affiliation_identifier_scheme = a["affiliationIdentifierScheme"]
-          scheme_uri = a["SchemeURI"]
```

Comment:
`scheme_uri = a["SchemeURI"]` here character `S` was capital, and XML metadata file contains the small case. Because of which `scheme_uri` was getting returned in the response for some properties. I have fixed this change and moved the variable at the top of the block.

Reply:
Great spot!